pylint/lint/pylinter.py

```diff
@@ -658,13 +658,14 @@ def check(self, files_or_modules: Sequence[str]) -> None:
         files_or_modules is either a string or list of strings presenting modules to check.
         """
         self.initialize()
+        if not files_or_modules and not self.config.from_stdin:
+            files_or_modules = (str(Path(".").resolve()),)
         if self.config.recursive:
             files_or_modules = tuple(self._discover_files(files_or_modules))
         if self.config.from_stdin:
             if len(files_or_modules) != 1:
-                raise exceptions.InvalidArgsError(
-                    "Missing filename required for --from-stdin"
-                )
+                print("Missing filename required for --from-stdin")
+                sys.exit(32)
```

Comment:
Is this necessary for the current change? A small thing, but I think print() goes to stdout and exceptions go to stderr, so it might be nicer the way it was.

Reply:
Got it, makes sense.